upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -692,6 +692,13 @@ func trimCommonPrefix(names []string) []string {
 		}
 	}
 
+	for i := range names {
+		_, err := strconv.Atoi(names[i])
+		if err == nil {
+			names[i] = "e" + names[i]
+		}
```

Comment:
why `e`? I think it might be confusing to someone glancing at the cluster spec that kops just created for them.

```yaml
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-west-2a-1
      name: e1
    - instanceGroup: master-us-west-2a-2
      name: e2
    - instanceGroup: master-us-west-2a-3
      name: e3
    memoryRequest: 100Mi
    name: main
```
How about `etcd-` or `etcd` ?

```yaml
  etcdClusters:
  - cpuRequest: 200m
    etcdMembers:
    - instanceGroup: master-us-west-2a-1
      name: etcd-1
    - instanceGroup: master-us-west-2a-2
      name: etcd-2
    - instanceGroup: master-us-west-2a-3
      name: etcd-3
    memoryRequest: 100Mi
    name: main
```

Reply:
I didn't have better inspiration at that time :).